src/v/kafka/server/errors.h

```diff
@@ -113,4 +114,48 @@ constexpr error_code map_topic_error_code(cluster::errc code) {
     return error_code::unknown_server_error;
 }
 
+constexpr error_code map_tx_errc(cluster::tx::errc ec) {
```

Comment:
wonder if this belongs in tx_err.h? kafka/server/errors.h depending on (internal) tx_err seems a bit odd.

Reply:
i think the other way around would be incorrect as it would make cluster submodule depend on kafka submodule (it already kind of does), this is the kafka protocol implementation detail on how to map a more internal error code to user facing protocol defined errors.